versioned_docs/version-v0.71/node-operators/visor.md

```diff
@@ -0,0 +1,56 @@
+---
+sidebar_position: 9
+title: Visor overview
+description: Running Visor allows for automatic software upgrades.
+---
+
+## What is Vega Visor?
+Vega Visor is a tool to help run all the parts of a validator or non validator node with data node in a standardised way allowing for automatic upgrades of the software.
+
+## Why would you use Visor?
+As the protocol develops, bugs are fixed and new features are added, the development team will release new versions of the software on [GitHub ↗](https://github.com/vegaprotocol/vega). 
+
+Without Visor, each node operator would need to be prompted to stop their nodes and upgrade their software when required. This is an error prone and tricky process as not all node operators work on the same time zone, and data node operators may not even be actively aware of an upgrade. Validator node operators who need to restart their nodes manually can lead to a delay in getting the network back up again. 
+
+Two things have been added to improve this process. 
+
+First, a vote takes place among consensus node operators to set a block height on which the network should be upgraded. 
+
+Second, Visor watches the status of a node and when it sees the node flag an upgrade, it automatically shuts it down, downloads the correct new version and restarts the node with the new version. This allows the node to be kept up to date without any need for node operator interaction.
```

Comment:
```suggestion
Second, Visor constantly monitors the node's status. If a protocol upgrade is proposed, Visor takes care of the process automatically. It will shut down the node, install the new version, and restart the node with the updated version. As a result, node operators don't have to do anything to ensure nodes are up-to-date.
```